modules/mule-teradata-connector/pages/index.adoc

```diff
@@ -32,11 +39,14 @@ Teradata Connector enables you to:
 * Run a single bulk update to perform multiple SQL requests.
 * Make Data Definition Language (DDL) requests.
 * Execute stored procedures and SQL scripts.
+* Support pooling profile configuration for database connection
```

Comment:
The last two are not really use cases. Maybe they deserve a separate list? For example:

```
The connector supports:
* connection pooling
* auto reconnects after timeouts
```

Reply:
Thanks Adam, I fixed that.